modules/database/main.tf

```diff
@@ -16,16 +17,19 @@ resource "aws_db_parameter_group" "postgres_db" {
 }
 
 resource "aws_db_instance" "pg_db" {
-  identifier                  = "${var.namespace}-${var.stage}-pg-db"
-  allocated_storage           = 20
-  engine                      = "postgres"
-  engine_version              = "15.4"
-  instance_class              = "db.t3.micro"
-  db_name                     = "postgres"
-  username                    = "postgres"
-  manage_master_user_password = true
-  parameter_group_name        = aws_db_parameter_group.postgres_db.name
-  skip_final_snapshot         = true
-  db_subnet_group_name        = var.subnet_group_id
-  vpc_security_group_ids      = [aws_security_group.rds_sg.id]
+  identifier                   = "${var.namespace}-${var.stage}-pg-db"
+  allocated_storage            = 20
+  engine                       = "postgres"
+  engine_version               = "15.4"
+  instance_class               = "db.t3.micro"
+  db_name                      = "postgres"
+  username                     = "postgres"
+  manage_master_user_password  = true
+  parameter_group_name         = aws_db_parameter_group.postgres_db.name
+  skip_final_snapshot          = true
+  db_subnet_group_name         = var.subnet_group_id
+  vpc_security_group_ids       = [aws_security_group.rds_sg.id]
+  storage_encrypted            = true
```

Comment:
we will need to confirm in our internal deploy if these changes cause database downtime